omnibus/config/software/datadog-agent.rb

```diff
@@ -21,6 +21,8 @@
 build do
   license :project_license
 
+  bundled_agents = ["process-agent", "security-agent", "system-probe"]
```

Comment:
Out of curiosity: the default value of `BUNDLED_AGENTS` is `["process-agent", "trace-agent", "security-agent"]` (in `tasks/utils.py`), but here `trace-agent` is not present.

Is there a specific reason for this difference? If yes, can a comment be added here to explain it?

Reply:
This is explained in the additional notes: the reason is that the impact in term of memory usage was significantly higher for the trace agent as it doesn't share many of the components the other agents use.

Reply:
Sounds good 👍. Would you mind adding a short explanation and/or link to this PR as a comment here, so that this context is not lost?